src/internal/connector/onedrive/collection_test.go

```diff
@@ -343,92 +345,6 @@ func (suite *CollectionUnitTestSuite) TestCollectionReadError() {
 	}
 }
 
-func (suite *CollectionUnitTestSuite) TestCollectionDisablePermissionsBackup() {
```

Comment:
this is now an e2e test in graph_connector_onedrive_test.go. The code it was exercising was changed such that it was no longer easy to test in a unit test